api/lib/domain/models/CertificationCandidate.js

```diff
@@ -22,13 +22,31 @@ const certificationCandidateValidationJoiSchema_v1_4 = Joi.object({
   sessionId: Joi.number().required().empty(null),
 });
 
+const certificationCandidateValidationJoiSchema_v1_5 = Joi.object({
+  firstName: Joi.string().required().empty(null),
```

Comment:
Je comprends pas bien le `.required().empty(null)`

J'ai testé sur https://joi.dev/tester
Avec : 
- schema: 
```
Joi.object({
  a: Joi.string().required().empty(null)
})
```
- Data To Validate:
```
{ 
 a: null
}
```
Je veux bien comprendre l'intention

Reply:
Surement pour harmoniser les messages d'erreur : 
Avec le `.empty(null)` => Validation Error: "a" is required
Si on enlève : `.empty(null)` => Validation Error: "a" must be a string

Reply:
de mémoire, je crois que le empty(null) fait la chose suivante : avant validation, si la chaîne est vide ( '' ), ça convertit la valeur en null, puis ça applique les règles de validation.
Ca évite de laisser passer les chaines vides je crois, à re-tester cela dit